test/Bugs/Issue3027.err

```diff
@@ -2,5 +2,5 @@ AGDA_FAILURE
 
 ret > ExitFailure 1
 out > An internal error has occurred. Please report this as a bug.
-out > Location of the error: src/full/Agda/TypeChecking/Reduce/Fast.hs:1340
+out > Location of the error: src/full/Agda/TypeChecking/Reduce/Fast.hs
```

Comment:
I guess we do not want the line number of the error to vanish here.  (This would make debugging next to impossible.)

Reply:
But then the golden value will change every time we add or remove lines
above this impossible!

Reply:
Ah, it is just removed from the .err file.  That's good!
(I thought it was gone in general.)